trajopt_sco/src/optimizers.cpp

```diff
@@ -476,8 +476,8 @@ OptStatus BasicTrustRegionSQP::optimize()
   results_.status = retval;
   results_.total_cost = vecSum(results_.cost_vals);
   LOG_INFO("\n==================\n%s==================", CSTR(results_));
-  callCallbacks(x_);
+  callCallbacks();
 
   return retval;
 }
-}
+}  // namespace sco
```

Comment:
https://github.com/ros-industrial-consortium/trajopt_ros/blob/01421b492743fc10dc651956062b5053259facd8/trajopt_sco/src/optimizers.cpp#L269

I also propose going ahead and cleaning up this class by removing the use `x_` by replacing it with `results_.x`